prinz/src/main/scala/pl/touk/nussknacker/prinz/model/repository/ModelRepository.scala

```diff
@@ -0,0 +1,8 @@
+package pl.touk.nussknacker.prinz.model.repository
+
+import pl.touk.nussknacker.prinz.model.Model
+
+trait ModelRepository {
+  def listModels(): Either[Exception, List[Model]]
+  def getModel(key: String): Either[Exception, Model]
+}
```

Comment:
you can use some vertical space to separate methods in trats, it's more readable then imo

Reply:
also some typed Exceptions should be used